javascript/javascript1/week2/homework.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+
+
+function getFullname(firstname,surname,useFormalName = false){ 
+    if (useFormalName){         
+        return `Lord.${firstname} ${surname}`
+    }else{
+       return `${firstname} ${surname}`
+    }
+    } 
+    
+    let firstName1=getFullname('Benjamin','Button');
+    let firstName2=getFullname('Benjamin','Button',true);
+    
+    console.log(firstName1);
+    console.log(firstName2);
+
+
+
+
+//Event application 
+
+let today=new Date();
+let todayIndexWeekday=today.getDay();
+const weekdays=[
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday']
+
+let eventdaysleft=10;
+let eventday = (todayIndexWeekday+ eventdaysleft) % 7
+console.log(weekdays[eventday]);
+
+
+
+//Weather wear
+
+function chooseClothesToWear(temperature){
+    if(temperature>0 && temperature<13){
+        console.log(`What about some jackets and a hat`);
+    }else if (temperature>13 && temperature< 20){
```

Comment:
If you call `chooseClothesToWear(13)` you will get the last `else` case with "wear shorts". To prevent that from happening you would have to use either `<=` in the first condition when using 13 or at the second one with `>=`.